src/main/java/mate/academy/Main.java

```diff
@@ -4,20 +4,26 @@
 import mate.academy.model.User;
 import mate.academy.service.AuthenticationService;
 import mate.academy.service.AuthenticationServiceImpl;
-import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
+        logger.info("Application version 1.0 started in production mode");
```

Comment:
The log message 'Application version 1.0 started in production mode' is informative, but consider including more context if applicable, such as environment details or startup parameters, to enhance clarity as suggested in the checklist.